ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-defaults/advanced-gas-fee-defaults.js

```diff
@@ -39,7 +47,7 @@ const AdvancedGasFeeDefaults = () => {
 
   const handleUpdateDefaultSettings = () => {
     if (isDefaultSettingsSelected) {
-      dispatch(setAdvancedGasFee(null));
+      dispatch(setAdvancedGasFee({ chainId, gasFeePreferences: undefined }));
```

Comment:
Looks like the type for `setAdvancedGasFee` has changed here - we should update the type in `actions.ts` accordingly

Reply:
Thanks for catching this, done in latest